src/transformers/models/bigscience176b/modeling_bigscience176b.py

```diff
@@ -69,25 +71,48 @@
 _TOKENIZER_FOR_DOC = "BigScience176BTokenizer"
 
 BIGSCIENCE176B_PRETRAINED_MODEL_ARCHIVE_LIST = [
-    "bigscience/BigScience176B",
+    "bigscience/bigscience-small-testing",
+    # "bigscience/BigScience176B",
     # See all BigScience176B models at https://huggingface.co/models?filter=bigscience176b
 ]
 
+# Utility functions below:
+
 def attention_mask_func(attention_scores, attention_mask):
         return attention_scores.masked_fill_(attention_mask, -10000.0)
 
+def transpose_tensor(x):
+    return x.transpose(0, 1).contiguous()
```

Comment:
Nitpick: not a huge fan of adding a method for a oneliner

Reply:
Agree! Should be fixed [here](https://github.com/thomwolf/transformers/pull/1/commits/a74f0f55b6d27df0cd8f901befdacf4dffc9b03b). Made sure also that the tests pass again